src/components/Customers.tsx

```diff
@@ -39,9 +33,12 @@ export const Customers = () => (
         viewport={{ once: true }}
         transition={{ duration: 0.4, delay: 0.3 }}
       >
-        <div className="shorebird-block-big-title mb-16 px-8 text-center sm:px-24 md:px-48">
-          Thousands of developers trust Shorebird to deliver millions of patches
-          each month.
+        <div className="shorebird-block-big-title mb-8 px-8 text-center sm:px-24 md:px-48">
+          Trusted by thousands of developers
```

Comment:
Maybe this just comes as "Trusted by developers" since we state the # in the next line.
```suggestion
          Trusted by developers
```